File: Leetcode/2221.cs
namespace Leetcode
{
    internal class _2221
    {
        public int TriangularSum(int[] nums)
        {
            var len = nums.Length - 1;
            var i = len - 1;

            while (i >= 0)
            {
                var j = 0;
                while (j <= i)
                {
                    nums[j] = (nums[j] + nums[j + 1]) % 10;
                    j++;
                }
                i--;
            }

            return nums[0];
        }
    }
}

File: Leetcode/go/2221.go
package main

func triangularSum(nums []int) int {
	n := len(nums) - 1
	i := n - 1
	for i >= 0 {
		for j := 0; j <= i; j++ {
			nums[j] = (nums[j] + nums[j+1]) % 10
		}
		i--
	}

	return nums[0]
}
